target/linux/realtek/image/Makefile

```diff
@@ -37,129 +37,15 @@ define Device/netgear_nge
   DEVICE_VENDOR := NETGEAR
 endef
 
-define Device/allnet_all-sg8208m
-  SOC := rtl8382
-  IMAGE_SIZE := 7168k
-  DEVICE_VENDOR := ALLNET
-  DEVICE_MODEL := ALL-SG8208M
-  UIMAGE_MAGIC := 0x00000006
-  UIMAGE_NAME := 2.2.2.0
+define Build/custom-uimage
```

Comment:
Didn't we get rid of this before? I think commit ff012f3a67eba3622265b2aaf38da6a0a4574021 ("build: uImage: allow override of default arguments" is supposed to allow for enough flexibility, no?

Reply:
Removed.